src/hooks/hooks.ts

```diff
@@ -0,0 +1,42 @@
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { useHistory } from 'react-router';
+import { useFixedLocation } from '../providers/RoutesProvider';
+
+export const useRefresh = () => {
+  const history = useHistory();
+  const { pathname } = useFixedLocation();
+
+  const refresh = useCallback((redirectPath) => {
+    history.replace("/");
```

Comment:
add comments

Reply:
removed. was accidentally added from the LSP, not needed in DM